PersonBook/PersonBook/PersonBook/Entity/Person.cs

```diff
@@ -1,25 +1,11 @@
-﻿
-using System.ComponentModel.DataAnnotations;
+﻿using System.ComponentModel.DataAnnotations;
 
 namespace PersonBook.Entity
 {
     class Person 
```

Comment:
Marking entities with `public` would be a better approach.

Reply:
Done